### PR TITLE
Simplify the `__init__.py` by moving the default to the schema

### PR DESCRIPTION
**Pull request by sbrunner**
Note that configuration.py, config.md, applications-md, poetry.lock :-) are generated files.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,6 +58,10 @@ All the provided commands:
 - `c2cciutils-download-applications`: Download the applications with version managed by Renovate, see below.
 - `c2cciutils-docker-versions-gen`: Generate the Docker package versions file (`ci/dpkg-versions.yaml`), see below.
 
+## Configuration
```
Comment (arnaud-morvan) [inline README.md:61]: Duplicated section "Configuration", another one already exists some lines below.
I propose to merge them.

Maybe add a link to applications.md too in section "Download applications"

File: config.md [hunk 2]
```diff
@@ -0,0 +1,83 @@
+# configuration
```
Comment (arnaud-morvan) [inline config.md:1]: This file seems not really friendly for humans.

We get a redirection "Refer to _#/definitions/checks_black_" just for one sentence : "The Black check configuration."
In such case, an example could help.

**Review by arnaud-morvan — APPROVED** [review]
Feel free to merge for the defaults, but the config.md file is not really friendly for now.

File: README.md (hunk 1 above)
Comment (sbrunner) [reply README.md:61]: +1 Thanks :-)

File: config.md (hunk 2 above)
Comment (sbrunner) [reply config.md:1]: Effectively, he can be improved...

**Pull request merged**